ocs_ci/deployment/networkpolicy.py

```
@@ -0,0 +1,49 @@
+
+    namespace = config.ENV_DATA.get(
+        "cluster_namespace", constants.OPENSHIFT_STORAGE_NAMESPACE
+    )
+    logger.info(f"Applying allow-all NetworkPolicy override to namespace '{namespace}'")
+
+    network_policy_data = templating.load_yaml(
+        constants.NETWORK_POLICY_ALLOW_ALL_OVERRIDE_TEMPLATE
+    )
+    network_policy_data["metadata"]["namespace"] = namespace
+
+    network_policy_file = tempfile.NamedTemporaryFile(
```

Ruff (0.16.1)

[warning] Use a context manager for opening files

(SIM115)